fe-nc-news/src/Components/RoutesContainer.jsx

```diff
@@ -1,13 +1,15 @@
 import { ArticleDetails } from "./ArticleDetails";
 import { Articles } from "./Articles";
 import { Route, Routes } from "react-router";
+import { Comments } from "./Comments";
 
 export const RoutesContainer = () => {
   return (
     <>
       <Routes>
         <Route path="/" element={<Articles />} />
         <Route path="/articles/:article_id" element={<ArticleDetails />} />
+        <Route path="/articles/:article_id/comments" element={<Comments />} />
```

Comment:
I'm getting errors when I go this path. Ideally it shouldn't be there and your comments should render on the single article page